jina/types/arrays/document.py

```diff
@@ -397,11 +397,82 @@ def reverse(self):
             hi_idx -= 1
         self._update_id_to_index_map()
 
-    def sort(self, key=None, *args, **kwargs):
```

Comment:
What is the point of this method?

Reply:
My idea was to keep the changes to the minimum of the already implemented sort method. I agree this method can be updated to accommodate top_k and reverse.